tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
     <string name="pref_dnscrypt_fallback_resolver_summ">Bootstrap resolver.
-        This is a normal, non-encrypted DNS resolver, that will be only used for one-shot queries
-        when retrieving the initial resolvers list, and only if the system DNS configuration doesn\'t work.
+        This is a normal, non-encrypted DNS resolver, only to be used for one-shot queries
+        when retrieving the initial list of resolvers, and only if the system DNS doesn\'t work.
         It will never be used if lists have already been cached.</string>
     <string name="pref_dnscrypt_fallback_resolver" translatable="false">bootstrap_resolvers</string>
-    <string name="pref_dnscrypt_ignore_system_dns_summ">Never let dnscrypt-proxy try to use the system DNS settings. Unconditionally use the fallback resolver..</string>
+    <string name="pref_dnscrypt_ignore_system_dns_summ">Never let dnscrypt-proxy try to use the system DNS settings. Unconditionally use the fallback resolver.</string>
     <string name="pref_dnscrypt_ignore_system_dns" translatable="false">ignore_system_dns</string>
 
-    <string name="pref_dnscrypt_filters_categ">FILTERS</string>
+    <string name="pref_dnscrypt_filters_categ">Filters</string>
 
-    <string name="pref_dnscrypt_forwarding_rules_categ">Forwarding rules</string>
-    <string name="pref_dnscrypt_forwarding_rules" translatable="false">forwarding_rules</string>
+    <string name="pref_dnscrypt_forwarding_rules_categ">Forwarding Rules</string>
+    <string name="pref_dnscrypt_forwarding_rules" translatable="false">Forwarding Rules</string>
     <string name="pref_dnscrypt_forwarding_rules_summ">Route queries for specific domains to a dedicated set of servers.</string>
-    <string name="pref_dnscrypt_import_forwarding_rules">Import forwarding rules</string>
-    <string name="pref_dnscrypt_import_forwarding_rules_summ">Import a forwarding rules file. You can select multiple files, duplicate lines will be removed.</string>
+    <string name="pref_dnscrypt_import_forwarding_rules">Import Forwarding Rules</string>
+    <string name="pref_dnscrypt_import_forwarding_rules_summ">Import a file of forwarding rules. You can select multiple files. Duplicated lines will be removed.</string>
     <string name="pref_dnscrypt_erase_forwarding_rules">Erase forwarding rules</string>
 
-    <string name="pref_dnscrypt_cloaking_rules_categ">Cloaking rules</string>
-    <string name="pref_dnscrypt_cloaking_rules" translatable="false">cloaking_rules</string>
+    <string name="pref_dnscrypt_cloaking_rules_categ">Cloaking Rules</string>
+    <string name="pref_dnscrypt_cloaking_rules" translatable="false">Cloaking Rules</string>
     <string name="pref_dnscrypt_cloaking_rules_summ">Cloaking returns a predefined address for a specific name.\n
         In addition to acting as a HOSTS file, it can also return the IP address of a different name. It will also do CNAME flattening..</string>
```

Comment:
```suggestion
        In addition to acting as a HOSTS file, it can also return the IP address of a different name.\nIt will also do CNAME flattening.</string>
```